test/app.test.js

```diff
@@ -0,0 +1,70 @@
+const expect = require('chai').expect;
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+const app = require('../app.js').app;
+const heroes = require('../app.js').heroes;
+
+describe('hero router', () => {
```

Comment:
should have used sinon all through the tests to mock data instead of using "real" data...

Reply:
all of your tests depend on each other, BAD PRACTICE